apps/web/src/pages/Pools/EditPoolPage/components/WorkTasksSection/WorkTasksSection.tsx

```diff
@@ -160,24 +161,16 @@ const WorkTasksSection = ({
               <div className="mb-6 grid gap-6 sm:grid-cols-2">
                 <RichTextInput
                   id="YourWorkEn"
-                  label={intl.formatMessage({
-                    defaultMessage: "English - Your work",
-                    id: "lb7SoP",
-                    description:
-                      "Label for the English - Your Work textarea in the edit pool page.",
-                  })}
+                  label={intl.formatMessage(processMessages.keyTasks)}
```

Comment:
Oops, this should be "Your work", not key tasks.

Reply:
It is a bit confusing because of the many changes over time to variable names and sections, but I believe this is correct. In the [issue description](https://github.com/GCTC-NTGC/gc-digital-talent/issues/14098#:~:text=Please%20change%20%22English%20%2D%20Your%20work%22%20to%20%22Common%20tasks%20in%20this%20role%20(English)%22), it asks to:

> Please change "English - Your work" to "Common tasks in this role (English)"

`processMessages.keyTasks` is now Common tasks in this role.